src/Switch/Switch.tsx

```diff
@@ -64,32 +58,22 @@ export default class Switch extends React.PureComponent {
       disabled,
     } = this.props;
 
-    const {hasOutline} = this.state;
-
     return (
       <button
         aria-checked={checked}
         aria-disabled={disabled}
         aria-labelledby={ariaLabelledby}
         aria-label={ariaLabel}
-        className={classnames(cssClass.CONTAINER, className)}
-        onMouseDown={() => this._onMouseDown()}
-        onMouseUp={() => this._onMouseUp()}
-        onFocus={this._onFocus}
-        onBlur={this._onBlur}
+        className={classnames(cssClass.CONTAINER, className, "flexbox", cssClass.BG,
```

Comment:
recommend using `.flexbox` as a mixin in the less file instead of adding it here directly, since this assumes that the flex styles are being fully imported (not just as reference) and isn't a pattern we should encourage